src/lib/draw/route/route_tool.ts

```diff
@@ -341,6 +328,18 @@ export class RouteTool {
     this.redraw();
   }
 
+  setHandlers = (eventHandler: EventHandler) => {
```

Comment:
This is from an older commit and not the new way of doing things in this PR, right? The way point mode/tool works looks fine, this is different...

Reply:
there was a reason I did it this way, and I have since made it work this way for Point Mode, so no this isn't a leftover from another commit. It might be less nice to do it this way. I don't the remember the reason I did this, it may have simply been that I thought it was nicer, but in any case it can't work this way for GeomMode so we may need to rethink or just let GeomMode be an exception